Simplified/Simplified-Info.plist

```diff
@@ -17,7 +17,7 @@
 	<key>CFBundlePackageType</key>
 	<string>APPL</string>
 	<key>CFBundleShortVersionString</key>
-	<string>2.2.2</string>
+	<string>2.2.3</string>
```

Comment:
Also a small thing, but it would be ideal to do version bumps outside of feature PRs like this: They don't have anything to do with the PR directly. It's fine to let it go this time though.

Reply:
Okay sounds good. Should it be a push directly on master after a release, or still PR'ed?

Reply:
I think small things like that can go right to master.